.travis.yml

```diff
@@ -76,6 +76,10 @@ before_install:
     elif node_version_lt '4.0' ; then npm_use_module 'supertest' '2.0.0'
     elif node_version_lt '6.0' ; then npm_use_module 'supertest' '3.4.2'
     fi
+  - |
+    # Configure debug package
+    if node_version_lt '6.0' ; then npm_use_module 'debug' '2.6.9'
```

Comment:
This won't work for users who are installing on those old version using `npm install compression`

Reply:
Yeah, realised after pushed that this won't work for non-dev dependencies. Are there any plans to update engine requirements for the package? Even v8 is no longer maintained in few days.

Reply:
Yea, as part of the express 5 work all these middlewares will be getting major bumps for things like the debug dep and their node support.

Reply:
Should I close this PR then or just update it with that expectation? Including tested versions on Travis.

Reply:
It's up to you what you would like to do :+1: some people do not like to have their PRs lingering, but if you are OK with that feel free to update your PR. Keep in mind that if you update other parts of the code in your PR like engine support that it would create a merge conflict by the time we merge, so if you want to keep it open, just leave it such that it assumes the master branch is updated.

Reply:
I don't mind it lingering. I reverted this change and will be waiting for the supported version bump. Feel free to close it otherwise.